src/scales/scale.logarithmic.js

```diff
@@ -138,26 +138,26 @@ module.exports = Scale.extend({
 				var meta = chart.getDatasetMeta(datasetIndex);
 				if (chart.isDatasetVisible(datasetIndex) && IDMatches(meta)) {
 					helpers.each(dataset.data, function(rawValue, index) {
-						var value = +me.getRightValue(rawValue);
+						var value = me._parseValue(rawValue);
 						// invalid, hidden and negative values are ignored
-						if (isNaN(value) || meta.data[index].hidden || value < 0) {
+						if (isNaN(value.min) || isNaN(value.max) || meta.data[index].hidden || value.min < 0 || value.max < 0) {
 							return;
 						}
 
 						if (me.min === null) {
```

Comment:
I would write this like so (same for max):
```js
if (me.min === null || me.min < value.min) {
  me.min = value.min;
}
```

Reply:
this code was before and i think we should avoid changing in this PR smth that is not related to float-bars logic. But the suggestion is reasonable for me

Reply:
+1. I like changing it as well since we're touching this code right now